src/youtube2zim/scraper.py

```diff
@@ -807,7 +807,7 @@ def update_metadata(self):
                 )
             else:
                 self.creator = _("Youtube Channels")
-        self.publisher = self.publisher or "Kiwix"
+        self.publisher = self.publisher
```

Comment:
<img width="958" alt="Screenshot 2024-03-07 164726" src="https://github.com/openzim/youtube/assets/101962582/910c2f94-8786-44be-9d7a-08bcd2c34d89">
I tested it, it works as expected.

Reply:
I'm not saying it's not working. It is, indeed, I can confirm this as well.

I'm questioning the fact that writing `self.publisher = self.publisher` provides any value. Could you explain me why we need to keep this statement?

Reply:
You're correct to question the statement `self.publisher = self.publisher`. I'm not entirely sure why it's included in the code.

Would you recommend removing it?

Reply:
The statement was `self.publisher = self.publisher or "Kiwix"`, meaning if `self.publisher` is not already set, then assign it the value `"Kiwix"`, otherwise keep original value. Now that you've removed this default assignment since it is done elsewhere, assigning the value of `self.publisher` to `self.publisher` is meaningless. I begin to seriously doubt you know what you are doing.